src/Type/Definition/BooleanType.php

```diff
@@ -14,33 +13,22 @@
 
 class BooleanType extends ScalarType
 {
-    /** @var string */
```

Comment:
we're missing type now

Reply:
Nope, it is defined in `Type`.

Reply:
but when the type of Type changes, it won't be detected here by SA

Reply:
https://github.com/webonyx/graphql-php/pull/906